### PR TITLE
Add new column to bpform and delivery form agg table

### PR DESCRIPTION
**Pull request by Rohit25negi**
Main task: remove ccs_record_tableau. Prerequisite for https://github.com/dimagi/commcare-hq/pull/22752

This Pr add the columns required for aggregation of ccs_Record. This should be deployed and let aggregated before the above Pr get merged.

### CODE REVIEW

File: custom/icds_reports/models/aggregate.py
Comment (emord) [inline custom/icds_reports/models/aggregate.py]: I mentioned this on a separate PR, but defaults are dangerous because they lock the table. can you change this, and re-run the migration to make sure a database default is not created. (I think you need to merge master and re-run the migration anyways due to migration conflicts

File: custom/icds_reports/utils/aggregation_helpers/birth_preparedness_forms.py
Comment (emord) [inline custom/icds_reports/utils/aggregation_helpers/birth_preparedness_forms.py]: Why is it necessary to use a `FULL JOIN` instead of a `LEFT JOIN`. I think that the UCR data source should contain every row necessary, so I'm not sure why we would need more rows from previous month table

File: custom/icds_reports/utils/aggregation_helpers/delivery_forms.py
Comment (emord) [inline custom/icds_reports/utils/aggregation_helpers/delivery_forms.py]: This will be a bit easier to read if you indent this line so that its clear its part of the previous line's calculation

**Review by emord — CHANGES_REQUESTED** [review]

File: custom/icds_reports/utils/aggregation_helpers/birth_preparedness_forms.py
Comment (Rohit25negi) [reply custom/icds_reports/utils/aggregation_helpers/birth_preparedness_forms.py]: Hey, here ucr is a filtered query result not the ucr actually which has only forms submitted this month. and this has indicators like `using_ifa` which are defined like https://docs.google.com/spreadsheets/d/1RDLNIHF3iRkiQtWyrQjKH6f1bStF_S_9woXeITiMJKg/edit#gid=551271206&range=A39. These kind of indicators which needs to consider prev month data similar to  https://github.com/dimagi/commcare-hq/blob/master/custom/icds_reports/utils/aggregation_helpers/growth_monitoring_forms.py

Comment (Rohit25negi) [reply custom/icds_reports/utils/aggregation_helpers/birth_preparedness_forms.py]: Does that makes sense? @emord

File: custom/icds_reports/models/aggregate.py
Comment (Rohit25negi) [reply custom/icds_reports/models/aggregate.py]: Removed the default 0 @emord

File: custom/icds_reports/utils/aggregation_helpers/delivery_forms.py
Comment (Rohit25negi) [reply custom/icds_reports/utils/aggregation_helpers/delivery_forms.py]: DOne

File: custom/icds_reports/utils/aggregation_helpers/birth_preparedness_forms.py
Comment (emord) [reply custom/icds_reports/utils/aggregation_helpers/birth_preparedness_forms.py]: Ah yeah. that makes sense. 

I notice the other one uses `FULL OUTER JOIN` instead of `FULL JOIN` are those equivalent?

**Review by emord — APPROVED** [review]
looks good from my side

File: custom/icds_reports/utils/aggregation_helpers/birth_preparedness_forms.py
Comment (Rohit25negi) [reply custom/icds_reports/utils/aggregation_helpers/birth_preparedness_forms.py]: yeah they are equivalent of each other.

**Comment by Rohit25negi** [comment]
@calellowitz  does this looks good to you also?

**Review by calellowitz — APPROVED** [review]
looks good. we just need to be careful about this and the other pr with conflicting migrations, once we merge one we cant merge the other until we fix the conflicts

**Pull request merged**
